api.planx.uk/modules/webhooks/service/sendNotification/schema.ts

```diff
@@ -75,9 +75,26 @@ export const s3SubmissionSchema = z.object({
   }),
 });
 
+export const flowStatusSchema = z.object({
+  body: z.object({
+    event: z.object({
+      data: z.object({
+        new: z.object({
+          id: z.string(),
+          status: z.string(),
+        }),
+      }),
+    }),
+  }),
+  query: z.object({
+    type: z.literal("flow-status"),
+  }),
+});
```

Comment:
Just taken a closer look at the code here to see what's causing the Zod union error on the Pizza.

There's nothing wrong with the code - pretty certain of it! 

Running locally (and in API tests on CI) this works as expected - I think it's actually a Vultr build issue maybe? The error message returned to Hasura doesn't include this schema as a member of the union. I'll recreate the pizza just now to see if this resolves it.

Reply:
Recreating Vultr gives the same error. Next thought - do we have some sort of env var issue here - is it pointing at the staging Hasura instance

Reply:
Nope, just checked out the Pizza using `docker inspect` and it's set to the expected value `HASURA_PLANX_API_URL=http://api:7002` 🤔

Reply:
Thanks for the second set of eyes here - it's a weird one !! 

Are you actually getting a succeeding `200` event locally? I have passing API tests, but failing local event similar to pizza.

Another thought - could it be a bad migration or error in the yaml?? I "updated" the event definition in the console at least once which seemed to correctly update `tables.yaml`? I'll try a docker destroy/recreate locally on this branch to trigger the migration again just to confirm

Reply:
Yeah I was getting past the validation middleware issue locally, and hitting the "staging application, skipping post to Slack" message which I was expecting to see.

There is an issue with the YAML docs here - there's an additional layer of `body` wrapping in the payload in the docs (issue not introduced by you!), and the endpoint is actually kebab-case in reality, not camelCase as in the docs. Was going to push up a small PR to address both of these points but happy to jump on a quick call to take a look here if helpful.

Reply:
Please see API doc fixes at https://github.com/theopensystemslab/planx-new/pull/3485